README.md

```diff
@@ -214,17 +216,14 @@ This means that the following will not trigger a sound:
 
 Please [open an issue][issue] if you want any other text events to play unique sounds.
 
-## WAV is the only sound fileformat playable on Windows
+## WAV is the only sound file format playable on Windows
 
 HexChat officially supports Linux and Windows, and uses different mechanisms to play sound files on each platform.
 
 On Windows, HexChat calls [`PlaySound`][PlaySound], which can process some kinds of [WAV][] files.
 
 [Audacity][] can be used to [import audio][import-audio], and then [export the audio][export-audio] in one of the following formats understood by [`PlaySound`][PlaySound]:
 
-* Signed 16-bit PCM
-* Signed 16-bit PCM
-* Signed 16-bit PCM
 * Signed 16-bit PCM

```

Comment:
All these were identical. Unless there was some reason for listing the same item four times, I removed the duplicates.

Reply:
Thank you. I meant to list 16-, 24-, 32-bit.